src/transaction.ts

```diff
@@ -253,6 +256,13 @@ export class Transaction implements TransactionStorageStructure {
       throw Error('firstRound must be a positive number');
     if (!Number.isSafeInteger(txn.lastRound) || txn.lastRound < 0)
       throw Error('lastRound must be a positive number');
+    if (
+      txn.extraPages != null &&
```

Comment:
I'm not sure this null check is necessary

Reply:
in my original commit it checks for undefined, which is necessary because extrapages is only defined in create tests. otherwise the error gets triggered because undefined is not an integer

Reply:
Right, I suppose `Number.isInteger` would return false for null, rending the null check a little redundant.

Reply:
its not redundant because we don't want it to return false for undefined. this test should only apply to cases where the parameter was passed in

Reply:
Great point!